benchmarks/dynamo/check_hf_bert_perf_csv.py

```diff
@@ -18,7 +18,8 @@ def check_hf_bert_perf_csv(filename):
         speedup = row["speedup"]
         # Reduced from 1.19 to 1.17, see https://github.com/pytorch/pytorch/issues/94687
```

Comment:
nit: should we delete earlier comments as well?

Reply:
Good point, let me delete the earlier comments and just leave the current one there.

Reply:
Done!